Sources/SwiftLanguageService/ParametersDocumentationExtractor.swift

```diff
@@ -0,0 +1,192 @@
+import Foundation
+import Markdown
+
+private struct ParametersDocumentationExtractor {
```

Comment:
Could you add the discussion in your PR about why we can’t (easily) use swift-docc here? I think that’s going to be a useful reference in the future as well.

Reply:
Should I add the PR link or the full discussion? 👀